Assets/LeapMotionModules/UI/Scripts/Constraints/RectConstraint.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Leap.Unity.Attributes;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.UI.Constraints {
+
+  public class RectConstraint : Linear2Constraint {
```

Comment:
What is the difference between a Linear2Constraint and a RectConstraint?  What other types of constraints could extend from Linear2Constraint that wouldn't be a RectConstraint?